include/deal.II/base/table.h

```diff
@@ -1499,6 +1501,204 @@ class Table<7,T> : public TableBase<7,T>
 
 
 
+/**
+ * A namespace for iterators for TransposeTable. TransposeTable is unique in
+ * that it stores entries in column-major (i.e., Fortran) order.
```

Comment:
I am not sure if this type of storage is a property unique to `TransposeTable`. Maybe rather "special"?

Do we really need to mention Fortran here?